src/overlays/actors/ovl_Bg_Ikana_Mirror/z_bg_ikana_mirror.c

```diff
@@ -4,7 +4,10 @@
  * Description: Stone Tower Temple - Mirror
  */
 
+// credits for documentation: Anon58, EllipticEllipsis
```

Comment:
We don't do credits in files, co-authoring the commit or mentioning it in the PR is the usual way we give credit.

Reply:
These are usually removed when I add co-authors. Did them by batch for this PR and this is the last batch. Should be in the next pushed commit.